src/azul/bigquery_reservation.py

```diff
@@ -0,0 +1,142 @@
+from typing import (
+    Optional,
+)
+
+import attr
+from google.cloud.bigquery_reservation_v1 import (
+
```

Comment:
afaik we don't need a new line.

Reply:
Yup, that a typo, thanks for catching it.